media-tv/mythtv/mythtv-30.0_p20190808.ebuild

```diff
@@ -0,0 +1,369 @@
+	myconf+=("$(use_enable vpx libvpx)")
+
+	if use debug; then
+		myconf+=("--compile-type=debug")
+		#myconf+=("--enable-debug") does nothing per sphery
+		myconf+=("--disable-stripping") # FIXME: does not disable for all files, only for some
+		myconf+=("--enable-valgrind") # disables timeouts for valgrind memory debugging
+	else
+		myconf+=("--compile-type=release")
+	fi
+
+	# Video Output Support
+	myconf+=("--enable-x11")
+	myconf+=("--enable-xrandr")
+	myconf+=("--enable-xv")
+	myconf+=("$(use_enable vdpau)")
+	myconf+=("$(use_enable vaapi)")
+
+	# Random stuff
+	myconf+=("$(use_enable zeroconf libdns-sd)")
+	myconf+=("$(use_enable libass)")
+
+	# Clean up DSO load times and other compiler bits
+	myconf+=("--enable-symbol-visibility")
+	myconf+=("--enable-pic")
+
+	if tc-is-cross-compiler ; then
+		myconf+=("--enable-cross-compile --arch=$(tc-arch-kernel)")
+		myconf+=("--cross-prefix=${CHOST}-")
+	fi
+
+	# Build boosters
+	has distcc ${FEATURES} || myconf+=("--disable-distcc")
+	has ccache ${FEATURES} || myconf+=("--disable-ccache")
+
+	einfo "Running ./configure ${myconf[@]}"
+	./configure \
+		--cc="$(tc-getCC)" \
+		--cxx="$(tc-getCXX)" \
+		--ar="$(tc-getAR)" \
+		--extra-cflags="${CFLAGS}" \
+		--extra-cxxflags="${CXXFLAGS}" \
+		--extra-ldflags="${LDFLAGS}" \
+		--qmake=$(qt5_get_bindir)/qmake \
+		${myconf[@]}
+}
+
+src_install() {
+	emake STRIP="true" INSTALL_ROOT="${D}" install
+	dodoc AUTHORS UPGRADING README
+	readme.gentoo_create_doc
+
+	insinto /usr/share/mythtv/database
+	doins database/*
+
+	newinitd "${FILESDIR}"/mythbackend.init-r2 mythbackend
+	newconfd "${FILESDIR}"/mythbackend.conf-r1 mythbackend
+	if use systemd; then
+		systemd_newunit "${FILESDIR}"/mythbackend.service-28 mythbackend.service
+	fi
+
+	dodoc keys.txt
+
+	keepdir /etc/mythtv
+	fowners -R mythtv /etc/mythtv
+	keepdir /var/log/mythtv
+	fowners -R mythtv /var/log/mythtv
+
+	insinto /etc/logrotate.d
+	newins "${FILESDIR}"/mythtv.logrotate.d-r4 mythtv
+
+	insinto /usr/share/mythtv/contrib
+	# Ensure we don't install scripts needing the perl bindings (bug #516968)
+	use perl || find contrib/ -name '*.pl' -exec rm -f {} \;
+	# Ensure we don't install scripts needing the python bindings (bug #516968)
+	use python || find contrib/ -name '*.py' -exec rm -f {} \;
+	doins -r contrib/*
+
+	# Install our mythfrontend wrapper which is similar to Mythbuntu's
+	if use wrapper; then
+		mv "${ED}/usr/bin/mythfrontend" "${ED}/usr/bin/mythfrontend.real" || die "Failed to install mythfrontend.real"
+		newbin "${FILESDIR}"/mythfrontend.wrapper mythfrontend
+		newconfd "${FILESDIR}"/mythfrontend.conf mythfrontend
+	fi
+
+	if use autostart; then
+		dodir /etc/env.d/
+		echo 'CONFIG_PROTECT="/home/mythtv/"' > "${ED}"/etc/env.d/95mythtv
+
+		insinto /home/mythtv
+		newins "${FILESDIR}"/bash_profile .bash_profile
+		newins "${FILESDIR}"/xinitrc-r1 .xinitrc
+	fi
+
+	# Make Python files executable
+	find "${ED}/usr/share/mythtv" -type f -name '*.py' | while read file; do
```

Comment:
This should probably be under `if use python; then`?

Reply:
Maybe. I going to leave it alone for now.

Reply:
You shouldn't. I didn't see any .py files under /usr/share in the source directory, or list of installed files even with python USE enabled. Seems like this whole section is useless, then?

Reply:
I found a lot of .py files under /usr/share/mythtv sub directories on my system. I moved mythtv to a backup directory and did a complete media-tv/mythtv emerge.  With python USE disabled, a lot of .py files are created in sub directories of /usr/share/mythtv .  They appear to be for support of some mythplugins. I am guessing that this is the reason mythtv must be _installed_ before mythplugins can be _compiled._  This code needs to be here until upstream fixes the packaging.

Then again this could just be my old system.

Reply:
> They appear to be for support of some mythplugins. I am guessing that this is the reason mythtv must be _installed_ before mythplugins can be _compiled._ This code needs to be here until upstream fixes the packaging.
> 

Hmm, indeed, I see them now.